File: python_programming_basics/conditional_statements_advanced/l_07_working_hours.py
current_hour = int(input())
week_day = input()

if week_day == 'Monday' or week_day == 'Tuesday' or week_day == 'Wednesday' or \
        week_day == 'Thursday' or week_day == 'Friday' or week_day == 'Saturday':
    if 10 <= current_hour <= 18:
        print('open')
    else:
        print('closed')

else:
    print('closed')

File: JavaScript Basics/03. Conditional Statements Advanced/l-07-Working-Hours.js
function isOfficeOpen(hour, day) {
    const workingDays = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
    
    if (workingDays.includes(day) && hour >= 10 && hour < 18) {
        console.log("open");
    } else {
        console.log("closed");
    }
}

isOfficeOpen(11, "Monday");
isOfficeOpen(19, "Friday");
isOfficeOpen(11, "Sunday");
